astroquery/desi/tests/test_module.py

```diff
@@ -0,0 +1,132 @@
+import astropy.io.votable
+import pytest
+import os
+import numpy as np
+import pyvo as vo
+
+from astropy.table import Table
+from astropy.io import fits
+from pyvo.dal import TAPResults
```

Comment:
why not use this similarly to TAPService as `vo.dal.TAPResults`? I'm OK either way, just wondering if there is a reason.